frequency.py

```diff
@@ -2,6 +2,11 @@
 Project Gutenberg """
 
 import string
+import pickle
+
+input_file = open('conandoyle.pickle', 'rb')
```

Comment:
These load statements should likely go at the bottom of the file, in `if __name__ == "__main__":`